hack/update-deps.sh

```diff
@@ -26,24 +26,12 @@ set -o errexit
 set -o pipefail
 set -o xtrace
 
-TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
-cd "${TESTINFRA_ROOT}"
-
+cd "$(git rev-parse --show-toplevel)"
 trap 'echo "FAILED" >&2' ERR
-# dep itself has a problematic testdata directory with infinite symlinks which
-# makes bazel sad: https://github.com/golang/dep/pull/1412
-# dep should probably be removing it, but it doesn't:
-# https://github.com/golang/dep/issues/1580
-rm -rf vendor/github.com/golang/dep/internal/fs/testdata
-# go-bindata does too, and is not maintained ...
-rm -rf vendor/github.com/jteeuwen/go-bindata/testdata
-# docker has a contrib dir with nothing we use in it, dep will retain the licenses
-# which includes some GPL, so we manually prune this. 
-# See https://github.com/kubernetes/steering/issues/57
-rm -rf vendor/github.com/docker/docker/contrib
-bazel run //:dep -- ensure -v "$@"
-rm -rf vendor/github.com/golang/dep/internal/fs/testdata
-rm -rf vendor/github.com/jteeuwen/go-bindata/testdata
-rm -rf vendor/github.com/docker/docker/contrib
+
+rm -rf vendor
+bazel run //:go -- mod tidy
+bazel run //:go -- mod vendor
```

Comment:
Can we make these:
```bash
GO111MODULE=on bazel run //:go -- mod tidy
GO111MODULE=on bazel run //:go -- mod vendor
```

or export `GO111MODULE=on` earlier in this script.

Reply:
Done